benchmarks/cugraph/pytest-based/bench_cugraph_uniform_neighbor_sample.py

```diff
@@ -86,26 +94,36 @@ def create_graph(graph_data):
             source="src",
             destination="dst",
             edge_attr="weight",
-            renumber=True,
+            legacy_renum_only=True,
         )
 
     else:
         raise TypeError(f"graph_data can only be str or dict, got {type(graph_data)}")
 
-    return G
+    return (G, num_verts)
 
 
 def create_mg_graph(graph_data):
     """
-    Create a graph instance based on the data to be loaded/generated.
+    Create a graph instance based on the data to be loaded/generated, return a
+    tuple containing (graph_obj, num_verts, client, cluster)
     """
+    n_devices = os.getenv("DASK_NUM_WORKERS", 4)
+    n_devices = int(n_devices)
+    # range starts at 1 to let let 0 be used by benchmark/client process
+    visible_devices = ",".join([str(i) for i in range(1, n_devices+1)])
+
     (client, cluster) = start_dask_client(
-        enable_tcp_over_ucx=False,
-        enable_infiniband=False,
-        enable_nvlink=False,
-        enable_rdmacm=False,
-        net_devices=None,
+        # enable_tcp_over_ucx=True,
+        # enable_infiniband=False,
+        # enable_nvlink=True,
+        # enable_rdmacm=False,
+        protocol="ucx",
+        rmm_pool_size="28GB",
+        CUDA_VISIBLE_DEVICES=visible_devices,
```

Comment:
No argument `CUDA_VISIBLE_DEVICES` in `start_dask_client`

Reply:
yes, thanks, fixed and ran the test to verify this code path works now.